mongo/requirements-dev.txt

```diff
@@ -0,0 +1,2 @@
+mock==2.0.0
+pytest
```

Comment:
I believe these can just be `datadog-checks-dev` now instead of declaring these two individually.